src/less/notice/ds6/notice-mixin.less

```diff
@@ -1,3 +1,4 @@
 .notice-mixin() {
+    @import "icon.less";
```

Comment:
Why was this added? Icon's have their own mixin.

Reply:
Yes, this change looks invalid. Please remove.